server/Makefile

```diff
@@ -0,0 +1,116 @@
+DYNAMODB_CONTAINER_NAME=dynamodb-local
+DYNAMODB_ADMIN_CONTAINER_NAME=dynamodb-admin
+ENDPOINT_URL=http://localhost:8000
+
+.PHONY: fmt
+fmt:
+	cargo fmt
+
+.PHONY: check
+check:
+	cargo fmt --check
+	cargo clippy --all-features
+	cargo d --no-deps --all-features
+
+.PHONY: run
+run:
+	USE_DYNAMODB=1 AWS_ENDPOINT_URL=${ENDPOINT_URL} cargo run
+
+.PHONY: test/e2e
+test/e2e:
+	AWS_ENDPOINT_URL=${ENDPOINT_URL} cargo t -- --ignored
+
+.PHONY: dynamodb
+.ONESHELL: dynamodb
+dynamodb:
```

Comment:
I don't think there's much value in this being a make script given it doesn't actually manage _any_ files. Lets just make this a shell script to start a local dynamodb with the appropriate tables instead?

Reply:
sure thing

Reply:
Looks like the Makefile is still in the PR — did you intend to remove it?

Reply:
I actually did not, can we instead gitignore it?